extensions/mssql/src/tableDesigner/tableDesigner.ts

```diff
@@ -19,7 +19,8 @@ export function registerTableDesignerCommands(appContext: AppContext) {
 			database: context.connectionProfile.databaseName,
 			isNewTable: true,
 			id: generateUuid(),
-			connectionString: connectionString
+			connectionString: connectionString,
+			accessToken: context.connectionProfile.options.azureAccountToken
```

Comment:
Also have you thought about how you're going to handle token expiration? @cssuh was looking into a more generic way of handling that for all connections - and it seems like if you just use a connection made with a connection provider you can hook into whatever that solution ends up looking like.

Reply:
yes, i will sync up with him offline.